meta-psa/recipes-security/psa-trusted-storage-linux/psa-trusted-storage-linux_1.0.0.bb

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2019 Arm Limited and Contributors. All rights reserved.
+#
+# SPDX-License-Identifier: MIT
+
+SUMMARY = "ARM Platform Security Architecture (PSA) Protected Storage Library"
+DEPENDS = ""
+HOMEPAGE = "https://github.com/ARMmbed/psa-storage"
+DESCRIPTION = "A Linux C library reference implementation of the PSA protected_storage.h API"
+SECTION = "libs"
+
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = " \
+    file://${WORKDIR}/git/LICENSE;md5=302d50a6369f5f22efdb674db908167a \
+    file://${WORKDIR}/git/apache-2.0.txt;md5=3b83ef96387f14655fc854ddc3c6bd57 \
+    "
+
+SRC_URI = "git://git@github.com/armmbed/psa_trusted_storage_linux.git;protocol=ssh;nobranch=1"
+SRCREV = "2be73f8a50eca10b45f952798e2100afd00a99f2"
+
+PV .= "+git${SRCPV}"
+S = "${WORKDIR}/git"
+
+FILES_${PN}_remove = "/bin/*"
```

Comment:
>> I think this line is unnecessary because ${PN}-dev appears in PACKAGES before ${PN}.

I'll investigate this statement for its implications. Investigation yesterday found that without this statement the psa-storage-example-app would be packaged into ${PN}.

I'm not clear what's being requesting here. Please proffer a specific commit an if it works in testing I'll accept it.

Reply:
My proposal is:
```suggestion
```
Since `${PN}-dev` comes before `${PN}` in `PACKAGES`, the line
```
FILES_${PN}-dev += "/bin/psa-storage-example-app"
```
should mean that the example app goes into `${PN}-dev` rather than `${PN}`... Although, like I wrote below, I don't think the example app should be in `${PN}-dev` anyway. I think we should  have something like:
```
PACKAGES =. "${PN}-examples "
FILES_${PN}-examples = "${bindir}/psa-storage-example-app"
```

Reply:
@diego-sueiro proposed a commit which I accepted. Please take a look.